test/e2e/podstartup_test.go

```diff
@@ -53,17 +52,32 @@ var _ = Describe("Pod startup checker", Ordered, ContinueOnFailure, func() {
 	})
 
 	It("should report unhealthy status when pods cannot be scheduled", func() {
-		By("Removing required labels from all nodes to prevent pod scheduling")
+		By("Removing pod creation permissions from cluster-health-monitor to prevent pod scheduling")
```

Comment:
Changed the mechanism used to prevent pods from being scheduled because AKS automatic prevents editing labels/taints on individual nodes.